docs/user-quick-start.md

```diff
@@ -141,14 +141,26 @@ enough resources:
    drenv is set up properly
    ```
 
-1. Install `clusteradm` tool. See
-   [Install clusteradm CLI tool](https://open-cluster-management.io/docs/getting-started/installation/start-the-control-plane/#install-clusteradm-cli-tool)
-   for the details.
-   Version v0.8.1 or later is reuired.
-
-1. Install `subctl` tool, See
-   [Submariner subctl installation](https://submariner.io/operations/deployment/subctl/)
-   for the details.
+1. Install the `clusteradm` tool
+
+   ```
+   curl -L https://raw.githubusercontent.com/open-cluster-management-io/clusteradm/main/install.sh | bash
+   ```
+
+   For more info see
+   [Install clusteradm CLI tool](https://open-cluster-management.io/docs/getting-started/installation/start-the-control-plane/#install-clusteradm-cli-tool).
+   Version v0.8.1 or later is required.
+
+1. Install the `subctl` tool and add it to your PATH
+
+   ```
+   curl -Ls https://get.submariner.io | bash
+   export PATH=$PATH:~/.local/bin
+   echo export PATH=\$PATH:~/.local/bin >> ~/.profile
```

Comment:
Instead of this, add the standard steps to install in /usr/local/bin/ and remove the ~/.local/bin/subctl.

Same for other subctl sections.

Reply:
This will work:

```
curl -Ls https://get.submariner.io | bash
sudo install .local/bin/subctl /usr/local/bin/
rm .local/bin/subctl
```

We will improve it later as part of #1493